### PR TITLE
RANCHER-977 - Set-up notifications for not started karate tests

### PR DESCRIPTION
**Pull request by vkapylou**
See: https://issues.folio.org/browse/RANCHER-977

### CODE REVIEW

File: pipelines/folioRancher/folioScheduledTesting/karateTestsScheduledPipeline.groovy [hunk 1]
```diff
@@ -61,9 +61,14 @@ pipeline {
...
-                    def jobParameters = getEnvironmentJobParameters('apply', okapiVersion, clusterName,
-                        projectName, prototypeTenant, folio_repository, folio_branch)
-                    spinUpEnvironmentJob = build job: spinUpEnvironmentJobName, parameters: jobParameters, wait: true, propagate: false
+                    try {
+                        def jobParameters = getEnvironmentJobParameters('apply', okapiVersion, clusterName,
+                            projectName, prototypeTenant, folio_repository, folio_branch)
+                        spinUpEnvironmentJob = build job: spinUpEnvironmentJobName, parameters: jobParameters, wait: true, propagate: false
+                    } catch (Exception new_ex) {
+                        println('Failed to create environment: ' + new_ex)
+                        slackNotifications.sendPipelineFailSlackNotification("#rancher_tests_notifications")
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioScheduledTesting/karateTestsScheduledPipeline.groovy:70]: Here you need to propagate the error as after slack notification, the send pipeline will continue execution of the following stages.

File: pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile [hunk 2]
```diff
@@ -61,16 +61,21 @@ ansiColor('xterm') {
...
+               string(name: 'CONFIG_TYPE', value: 'testing'),
+               booleanParam(name: 'LOAD_REFERENCE', value: true),
+               booleanParam(name: 'LOAD_SAMPLE', value: true),
+               booleanParam(name: 'CONSORTIA', value: true),
+               booleanParam(name: 'GREENMAIL', value: false),
+               string(name: 'MEMBERS', value: '')]
+        } catch (Exception new_ex) {
+          println('Failed to provision environment: ' + new_ex)
+          slackNotifications.sendPipelineFailSlackNotification("#rancher_tests_notifications")
+        }
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile:78]: Here you need to propagate the error as after slack notification, the send pipeline will continue execution of the following stages.

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: pipelines/folioRancher/folioScheduledTesting/karateTestsScheduledPipeline.groovy (hunk 1 above)
Comment (vkapylou) [reply pipelines/folioRancher/folioScheduledTesting/karateTestsScheduledPipeline.groovy]: I've added the "throw new Exception"

File: pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile (hunk 2 above)
Comment (vkapylou) [reply pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile]: I've added the "throw new Exception"

**Review by guram-jalaghonia — APPROVED** [review]
last changes seems ok and workable for me

**Review by OHaimanov — APPROVED** [review]

**Review by eldiiar-duishenaliev — APPROVED** [review]
Thanks for fixing all previously commented issues.

**Pull request merged**